main.py

```diff
@@ -2162,12 +2162,12 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
 
                 if len(disenchants) == 1:
                     buytext = " (bounty filled)" if ordersFilled > 0 else ""
-                    self.message(channel, "Successfully disenchanted waifu %d%s and added %d points to %s's account" % (
+                    self.message(channel, "Successfully disenchanted waifu %d%s. Added %d points to %s's account" % (
```

Comment:
Idea for further shortening: `Successfully disenchanted waifu %d%s. %s gained %d points."

"gained points" is about half as long as "Added points to 's account".